jaseci_serv/jaseci_serv/svc/prometheus/config.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+PROMON_CONFIG = {
```

Comment:
@amadolid Let's brainstorm around this -- I think we should build a set of APIs under jsorc that turn on/off and configure each of the services. I think this config based API interface is not super intuitive and kinda hard to use.